src/mame/pc/sis630.cpp

```diff
@@ -17,14 +17,14 @@ TODO (main):
 - PS/2 mouse is unstable, worked around by disabling and using a serial mouse instead.
 
 TODO (usability, to be moved in a SW list):
-- windows xp sp3: tests HW then does an ACPI devtrap write ($48), will eventually BSoD with
+- Windows XP SP3: tests HW then does an ACPI devtrap write ($48), will eventually BSoD with
```

Comment:
Lowercase was intentional: all these entries needs to be ported over to SW list anyway ...

Reply:
Fixed!